app/pages/_preview.tsx

```diff
@@ -30,7 +32,7 @@ const Page = () => {
 
 export default Page;
 
-const chartState = {
+const chartState = migrateConfiguratorState({
```

Comment:
👍 so it was not an immer error after all 😅

Reply:
I added an issue for in the future check this problem #1525